lang/lang_en_es.txt

```diff
@@ -306,6 +350,14 @@
 "Print time"
 "Tiempo de imp.:"
 
+#MSG_FS_ACTION c=10
+"FS Action"
+"FS accion"
+
+#c=18
+"FS v0.4 or newer"
+"FS v0.4 o +nueva"
```

Comment:
typo "+"?

Reply:
+ = mas ;-)

Reply:
Also here we could reuse existing messages to save some space